src/desktop/email.rs

```diff
@@ -167,6 +168,13 @@ impl EmailRequest {
         self
     }
 
+    /// Sets the activation token.
+    #[must_use]
+    pub fn activation_token<'a>(mut self, activation_token: impl Into<Option<&'a str>>) -> Self {
```

Comment:
Do we expose that everywhere else as well?

Reply:
Nope.

Reply:
Then, we should for consistency or mark it as as private with a todo item saying we should handle it once we support passing activation